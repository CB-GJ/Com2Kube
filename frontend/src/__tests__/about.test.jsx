import React from "react"
import renderer from "react-test-renderer"
import About from "../pages/about/about"

it("renders correctly", () => {
  const wrapper = renderer.create(<About />).toJSON()
  expect(wrapper).toMatchSnapshot()
})
