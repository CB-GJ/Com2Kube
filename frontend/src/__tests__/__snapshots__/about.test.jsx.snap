// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="makeStyles-body-2"
  >
    <div
      className="MuiContainer-root MuiContainer-maxWidthLg"
    >
      <div
        className="sc-AykKD gDyBzM"
      >
        <div
          className="sc-AykKE bChJxL"
        >
          <img
            alt="Claire"
            className="sc-AykKC gSFPIp makeStyles-avatarImg-3"
            src="test-file-stub"
          />
          <span>
            <b>
              Claire
            </b>
             • 
            about.frontendDev
          </span>
        </div>
        <div
          className="sc-AykKE bChJxL"
        >
          <img
            alt="Gab"
            className="sc-AykKC gSFPIp makeStyles-avatarImg-3"
            src="test-file-stub"
          />
          <span>
            <b>
              Gabriel
            </b>
             • 
            about.backendDev
          </span>
        </div>
      </div>
      <h4
        className="MuiTypography-root MuiTypography-h4 MuiTypography-alignJustify"
      >
        about.teamHeader
      </h4>
      <p>
        about.item1
      </p>
      <p>
        about.item2
      </p>
      <p>
        about.item3
      </p>
      <p>
        about.item4
      </p>
    </div>
    <div
      className="makeStyles-container-5"
    >
      <div
        className="sc-AykKD gDyBzM"
      >
        <img
          alt="FAQ"
          className="makeStyles-img-4"
          src="test-file-stub"
        />
      </div>
      <div
        className="makeStyles-item-6"
      >
        <h5
          className="MuiTypography-root MuiTypography-h5"
        >
          about.faq1
        </h5>
        <p>
          about.faq1Text1
        </p>
        <p>
          about.faq1Text2
        </p>
      </div>
    </div>
    <div
      className="makeStyles-container-5"
    >
      <div
        className="makeStyles-item-6"
      >
        <h5
          className="MuiTypography-root MuiTypography-h5"
        >
          about.faq2
        </h5>
        <p>
          about.faq2Text1
        </p>
        <p>
          about.faq2Text2
        </p>
      </div>
      <div
        className="sc-AykKD gDyBzM"
      >
        <img
          alt="solution"
          className="makeStyles-img-4"
          src="test-file-stub"
        />
      </div>
    </div>
  </div>
</div>
`;
